library/packages/src/lib/y2packager/licenses_fetchers/dummy.rb

```diff
@@ -0,0 +1,49 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2018 SUSE LLC, All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+
+require "y2packager/licenses_fetchers/base"
+
+module Y2Packager
+  module LicensesFetchers
+    # FIXME: Finish implementation
+    class Dummy < Base
```

Comment:
what is goal of this Dummy implementation? Should it act as NullObject pattern?

Reply:
As commented before, just a "transitional" class that should be replaced with something better for SP1.

Reply:
I wanted to keep the same internal API (the license asks its fetcher always about the content). As an alternative, we could use a different approach and just keep a `@content` variable in the licenses that would be use when no fetcher is defined at all.

Reply:
It is fine this way. I just maybe call it `Static` and not dummy. It is static fetcher, that have always same content.